meetings/2023-02-23.md

```diff
@@ -0,0 +1,113 @@
+URL: https://github.com/solid/notifications/pull/147
+
+>ACTION: SC to follow-up on creating issue for `filter`
+
+>* SC: To do.
+>* SC: I think we said this is next release material in earlier meetings, so I'll mark it as 0.3 when I create the issue.
+
+* SC: https://github.com/solid/notifications/issues/159
+
+>ACTION: SC to do editorial update to clarify the diffs better.
+
+* SC: To do.
+
+
+
+### Rename Subscription Client
+URL: https://github.com/solid/notifications/issues/142
+
+>ACTION: eP out of scope communication between different products and interop, taking into account possible Discovery Client extraction.
+
+* SC: To do.
+* SC: Ack 0.3 milestone.
+* eP: https://github.com/solid/notifications/issues/152 for revisiting expected interactions.
+* SC: So, add DiscoveryClient to #classes-of-products , remove (or change) SubscriptionClient->ResourceServer and add (or change to) DiscoveryClient->ResourceServer to https://solid.github.io/notifications/protocol#interoperability 
+* SC: DiscoveryClient as a product doesn't quite jump out. (I still think we don't need to separate this as a separate product from SubscriptionClient.)
+* SC: I do like the idea that DiscoveryClient can be extended to have a SubscriptionClient (or the other way, SC would build DC as well.)
+* eP: Discovery Client also discovers the Subscription Service (run by Subscription Server), not only Notification Channel.
+
+ACTION: JM to review 142 and 152.
+
+
+### Define unsubscribing
+URL: https://github.com/solid/notifications/issues/145
+
+* SC: ...
+* eP: Need information on how to unsubscribe. If someone created a subscription, and then whoever the receiver considers it unwanted should be able to unsubscribe. Each notification could do that.
+* SC: So, NotificationReceiver and SubscriptionClient are two products (and uses cases) that could potentially unsubscribe?
+* eP: Could look at it two ways. NR can unsubscribe or NR can also out of scope give info to SC that can follow up with the SC. I like SC being responsible with that. NR would not have to interact.
+* SC: Seems whatever conforms to SC would do it in the end.
+* eP: Comes down to authn/z at some point. SC would unsubscribe especially it initially authenticates.
+* SC: Notification may only need to know the notification-channel that the notification is part of. SC can use that info.
+* eP: 
+* SC: Let's look at two cases, sendTo and receiveFrom, for receiveFrom it seems that you have application doing both NR and SC,
+* eP: in receiveFrom the NR can just disconnect, so it's mostly about sendTo where NR might need information to unsubscribe.
+* SC: For any notification channel that is active server needs to have consistent way to unsubscribe. 
+* eP: If you just disconnect like on mobile device, a loop (like in the sequence diagram), you could reconnect. with capability based channel, when done, i can subscribe, no one can then reconnect to existing channel because it is no longer active.
```

Comment:
```suggestion
* eP: Need information on how to unsubscribe. If someone created a subscription, and then whoever the receiver considers it unwanted, they should be able to unsubscribe. Each notification could do that.
* SC: So, `NotificationReceiver` and `SubscriptionClient` are two products (and use cases) that could potentially unsubscribe?
* eP: Could look at it two ways. `NR` can unsubscribe or `NR` can also out of scope give info to `SC` that can follow up with the `SC`. I like `SC` being responsible with that. `NR` would not have to interact.
* SC: Seems whatever conforms to `SC` would do it in the end.
* eP: Comes down to authn/z at some point. `SC` would unsubscribe especially it initially authenticates.
* SC: Notification may only need to know the notification-channel that the notification is part of. `SC` can use that info.
* eP: 
* SC: Let's look at two cases, `sendTo` and `receiveFrom`. For `receiveFrom`, it seems that you have application doing both `NR` and `SC`.
* eP: In `receiveFrom` the `NR` can just disconnect, so it's mostly about `sendTo` where `NR` might need information to unsubscribe.
* SC: For any notification channel that is active, server needs to have consistent way to unsubscribe. 
* eP: If you just disconnect like on mobile device, a loop (like in the sequence diagram), you could reconnect. With capability based channel, when done, I can subscribe; no-one can then reconnect to existing channel, because it is no longer active.
```